gitlab_runner/tests/conftest.py

```diff
@@ -52,7 +52,7 @@ def teardown():
         raise Exception("gitlab_runner container timed out!")
 
     # run pre-test commands
-    for i in xrange(100):
+    for i in range(100):
```

Comment:
Since this line is already being modified, this could be an opportunity to change `i` to `_` as well (since it doesn't seem to be used inside the loop).

```python
for _ in range(100):
```

Reply:
I prefer to limit this change to python 3 related changes only

Reply:
You think these are extensive modifications?

Reply:
Please don't mischaracterize my statements